### ISSUE TITLE
Code about Point-MAE

### ISSUE THREAD

**Issue by ZewLin** [Ivan-Tang-3D/Point-PEFT#2]
Hello dear authors, thanks for your amazing work!
I find the current code use Point-M2AE as backbone, can you release the code use Point-MAE as backbone?
Thanks!!!

**Comment by Ivan-Tang-3D** [comment]
Thanks for your appreciation. Since I am busy these few days, I will release the related code for Point-MAE in the near future. Once it is released, I will notify you.

**Comment by Ivan-Tang-3D** [comment]
> Hello dear authors, thanks for your amazing work! I find the current code use Point-M2AE as backbone, can you release the code use Point-MAE as backbone? Thanks!!!

Thanks for your appreciation. Since I am busy these few days, I will release the related code for Point-MAE in the near future. Once it is released, I will notify you.

**Comment by ZewLin** [comment]
> > Hello dear authors, thanks for your amazing work! I find the current code use Point-M2AE as backbone, can you release the code use Point-MAE as backbone? Thanks!!!
> 
> Thanks for your appreciation. Since I am busy these few days, I will release the related code for Point-MAE in the near future. Once it is released, I will notify you.

Thanks for your reply!!!

**Comment by ZewLin** [comment]
I have another question, did you reproduce the full-parameter finetuning  in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!

**Comment by Ivan-Tang-3D** [comment]
> I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!

Yes, the results I reproduced are close to the performance as its paper mentioned.

**Comment by ZewLin** [comment]
> > I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!
> 
> Yes, the results I reproduced are close to the performance as its paper mentioned.

OK,thanks! I am also research in point pre-training, can we add a contact information to communicate in detail? My wechat account is wxid_swxw9jnfctiu22.  Thanks again for your reply and help!!!

**Comment by Ivan-Tang-3D** [comment]
> > > I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!
> > 
> > 
> > Yes, the results I reproduced are close to the performance as its paper mentioned.
> 
> OK,thanks! I am also research in point pre-training, can we add a contact information to communicate in detail? My wechat account is wxid_swxw9jnfctiu22. Thanks again for your reply and help!!!

Sorry, the wechat account displays unavailable.

**Comment by ZewLin** [comment]
> > > > I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!
> > > 
> > > 
> > > Yes, the results I reproduced are close to the performance as its paper mentioned.
> > 
> > 
> > OK,thanks! I am also research in point pre-training, can we add a contact information to communicate in detail? My wechat account is wxid_swxw9jnfctiu22. Thanks again for your reply and help!!!
> 
> Sorry, the wechat account displays unavailable.

Oh sorry , it's my mistake. Could your retry the account 19550151227? Thanks!

**Comment by ZewLin** [comment]
> > > > > I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!
> > > > 
> > > > 
> > > > Yes, the results I reproduced are close to the performance as its paper mentioned.
> > > 
> > > 
> > > OK,thanks! I am also research in point pre-training, can we add a contact information to communicate in detail? My wechat account is wxid_swxw9jnfctiu22. Thanks again for your reply and help!!!
> > 
> > 
> > Sorry, the wechat account displays unavailable.
> 
> Oh sorry , it's my mistake. Could your retry the account 19550151227? Thanks!

@Ivan-Tang-3D Hello, I'm sorry to bother you. Is this account available?

**Comment by Ivan-Tang-3D** [comment]
> > > > > > I have another question, did you reproduce the full-parameter finetuning in modelnet and scan dataset based on Point-MAE and reach to the result as mentioned in paper? Looking forward to your reply! Thanks again!
> > > > > 
> > > > > 
> > > > > Yes, the results I reproduced are close to the performance as its paper mentioned.
> > > > 
> > > > 
> > > > OK,thanks! I am also research in point pre-training, can we add a contact information to communicate in detail? My wechat account is wxid_swxw9jnfctiu22. Thanks again for your reply and help!!!
> > > 
> > > 
> > > Sorry, the wechat account displays unavailable.
> > 
> > 
> > Oh sorry , it's my mistake. Could your retry the account 19550151227? Thanks!
> 
> [@Ivan-Tang-3D](https://github.com/Ivan-Tang-3D) Hello, I'm sorry to bother you. Is this account available?

Yeah, It is right.
